internal/namespaces/vpc/v1/custom_private_network.go

```diff
@@ -1,326 +0,0 @@
-package vpc
-
-import (
-	"context"
-
-	"github.com/scaleway/scaleway-cli/v2/internal/core"
-	"github.com/scaleway/scaleway-sdk-go/api/baremetal/v1"
-	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
-	"github.com/scaleway/scaleway-sdk-go/api/lb/v1"
-	"github.com/scaleway/scaleway-sdk-go/api/rdb/v1"
-	"github.com/scaleway/scaleway-sdk-go/api/redis/v1"
-	"github.com/scaleway/scaleway-sdk-go/api/vpc/v1"
-	"github.com/scaleway/scaleway-sdk-go/api/vpcgw/v1"
-	"github.com/scaleway/scaleway-sdk-go/scw"
-)
-
-func privateNetworkGetBuilder(c *core.Command) *core.Command {
```

Comment:
Is this function and set of features migrated to v2 as well?

Reply:
For a matter of time, I'll do it later in another PR